packages/core/src/connectors/index.ts

```diff
@@ -19,6 +20,12 @@ const loadConnectors = async () => {
     return cachedConnectors;
   }
 
+  const {
+    values: { additionalConnectorPackages },
+  } = envSet;
+
+  const connectorPackages = [...defaultConnectorPackages, ...additionalConnectorPackages];
+
```

Comment:
curious why move this part here, can't find a lot of difference.

Reply:
We use `env-set` to manage the env values and we can only use this when the `env-set` is loaded.